src/clr-angular/utils/drag-and-drop/draggable.spec.ts

```diff
@@ -6,32 +6,83 @@
 import {Component} from "@angular/core";
 import {TestBed} from "@angular/core/testing";
 import {By} from "@angular/platform-browser";
+import {NoopAnimationsModule} from "@angular/platform-browser/animations";
+
+import {ClrIconModule} from "../../icon/icon.module";
+import {DomAdapter} from "../dom-adapter/dom-adapter";
+import {ClrDragAndDropModule} from "./drag-and-drop.module";
 import {ClrDragHandle} from "./drag-handle";
 import {ClrDraggable} from "./draggable";
-import {ClrDragAndDropEventBus} from "./providers/drag-and-drop-event-bus";
 import {ClrDragEventListener} from "./providers/drag-event-listener";
+import {MOCK_DRAG_EVENT_LISTENER_PROVIDER} from "./providers/drag-event-listener.mock";
 import {ClrDragHandleRegistrar} from "./providers/drag-handle-registrar";
+import {ClrDraggableSnapshot} from "./providers/draggable-snapshot";
+import {ClrGlobalDragMode} from "./providers/global-drag-mode";
```

Comment:
For all new tests, the convention is to add the test components after the imports and then start testing those components. Could you move your test components right after the imports please?